.env.example

```
@@ -53,8 +53,16 @@ MASTER_KEY=""
 # Enabled by docker-compose.local.yml; leave disabled here to avoid accidental provisioning.
 HOMELAB_DEV_SEED="false"
 
-# Start management profile services (agent, socket-proxy)
-COMPOSE_PROFILES="management"
+# Start management profile services (agent, socket-proxy) and Pocket ID for local OIDC dev.
+# Remove "oidc" if you don't need the local Pocket ID instance.
+COMPOSE_PROFILES="management,oidc"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)